plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -104,9 +116,12 @@ bool EuroliteProFactory::DeviceAdded(
       }
       is_mk2 = true;
     } else {
-      OLA_INFO << "Connected FTDI device could be a Eurolite "
-               << "USB-DMX512-PRO MK2 but was ignored, because "
-               << ENABLE_EUROLITE_MK2_KEY << " was false.";
+      OLA_INFO << "Connected FTDI device with serial " << info.serial
+               << " could be a Eurolite USB-DMX512-PRO MK2 but was "
+               << "ignored, because "
+               << ENABLE_EUROLITE_MK2_KEY << " was false and "
+               << "its serial number was not in the "
+               << "configuration.";
```

Comment:
Maybe something like this is a bit clearer?
```suggestion
               << "its serial number was not listed "
               << "specifically in the configuration.";
```

Or better yet tell them which key they can add it under!

Reply:
Done thanks